scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -314,7 +314,14 @@ monitor_sst_progress() {
     fi
 
     if [[ ${current_timeout} -eq  ${timeout} ]]; then
+      wsrep_log_error "******************* FATAL ERROR ********************** "
       wsrep_log_error "Killing SST ($pid) with SIGKILL after stalling for ${current_timeout} seconds"
+      wsrep_log_error "This message is related to sst-idle-timeout configuration variable."
+      wsrep_log_error "It's current value is $WSREP_SST_IDLE_TIMEOUT seconds."
+      wsrep_log_error "PXC node didn't receive any data for last ${current_timeout} seconds during State Transfer (SST)."
+      wsrep_log_error "The most probable reason is broken network connectivity between Donor and Joiner (this) node."
+      wsrep_log_error "Check the network connection and restart the Joiner node."
```

Comment:
Can we rephrase it to something like?

_The SST process on the joiner (this node) did not receive any data from the donor despite waiting for ${current_timeout} seconds (defined by the sst-idle-timeout variable). This could be due to a broken network connectivity between the donor and the joiner (this node). Consider checking the network connection and restarting the Joiner node._

@patrickbirch Any suggestions?

Reply:
After waiting for the ${current_timeout} seconds (defined by the sst-idle-timeout variable), the SST process on the joiner (this node) has not received any data from the donor. This error could be caused by broken network connectivity between the donor and the joiner (this node). Check the network connection and restart the joiner node.